markdown/org/docs/patterns/noble/options/dartPosition/en.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Dart position"
+---
+
+The **Dart position** option controls whether to create the princess seam at the shoulder or armhole.
+
+## Effect of this option on the pattern
```

Comment:
Don't add this title. It will be auto-added if it's missing along with the image. But should not exist without the image